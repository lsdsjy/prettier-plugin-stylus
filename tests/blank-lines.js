run_spec(__dirname, { tabWidth: 4 });

run_spec(__dirname, {
  tabWidth: 4,
  parserOptions: {
    collapseBlankLines: true
  }
}, ["collapsed"]);
