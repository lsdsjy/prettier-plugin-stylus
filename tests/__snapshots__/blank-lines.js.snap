// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blank-lines 1`] = `
"body {
\tcolor: red

\tbackground: blue

\tborder: 1px solid black
}"
`;
